### DISCUSSION TITLE
moving pub from one community to another?

### DISCUSSION THREAD

**Discussion by RVirmoors** [knowledgefutures/pubpub#2832] [Support]
hello

i wrote a conference paper but mistakenly added it to a random community (learning.pubpub.org) rather than the conference domain. Is there a way to move it, so it can be published into the conference’s proceedings?

**Comment by gabestein** [comment]
Hello. Yes, we could move it for you. Please send a note to hello@pubpub.org with a link to the Pub and the community you'd like to move it to. Please make sure the email comes from an address of an admin in the community you would like to add it to.
